rest/admin_api.go

```diff
@@ -403,6 +403,17 @@ func (h *handler) handleGetRawDoc() error {
 		return err
 	}
 
+	if h.db.Options.UserXattrKey != "" {
+		rawUserXattrVal := doc.RawUserXattr
+		if len(rawUserXattrVal) == 0 {
+			rawUserXattrVal = []byte("null")
+		}
+		rawBytes, err = base.InjectJSONPropertiesFromBytes(rawBytes, base.KVPairBytes{Key: h.db.Options.UserXattrKey, Val: rawUserXattrVal})
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
This seems a bit mis-representative, given that the xattr in the sync function is nested under "meta", and this wouldn't.

Also what would happen if the doc already contains a property matching the xattr key (e.g. `"channels"`)?

Reply:
Thats a fair point... Similarly a user could have a 'meta' field in their doc though.